app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1099,6 +1099,49 @@ class PatientRepositoryAndroidTest {
         .dispose()
   }
 
+  @Test
```

Comment:
As discussed, please inject clock to all the places where you are instantiating time or date.
Feel free to do this in your next PR.